src/modules/ProfileLocation.rb

```diff
@@ -137,8 +137,7 @@ def Process
         end
         if fs
           AutoinstConfig.scheme = "device"
-          AutoinstConfig.host = fs.blk_devices.first.name
```

Comment:
Could `fs.blk_devices.first` be nil? For instance, when using DASD devices only (or something like that).